bool.go

```diff
@@ -0,0 +1,45 @@
+package nulltypes
+
+import (
+	"database/sql/driver"
+	"encoding/json"
+)
+
+//NullBool is a wrapper around bool
+type NullBool struct {
+	Bool  bool
+	Valid bool
+}
+
+// Bool method to get NullBool object from bool
+func Bool(Bool bool) NullBool {
+	return NullBool{Bool, true}
+}
+
+//MarshalJSON method is called by json.Marshal,
+//whenever it is of type NullBool
+func (x *NullBool) MarshalJSON() ([]byte, error) {
+	if !x.Valid {
+		return []byte("null"), nil
+	}
+	return json.Marshal(x.Bool)
+}
+
+// satisfy the sql.scanner interface
+func (x *NullBool) Scan(value interface{}) error {
+	rt, ok := value.(bool)
+	if ok {
+		*x = NullBool{rt, true}
+	} else {
+		*x = NullBool{rt, false}
+	}
+	return nil
+}
+
+// satifies the driver.Value interface
```

Comment:
The first word of comment should be a function name.

Reply:
These are not for the user to call, these are just implementing an interface.